src/index.ts

```diff
@@ -52,6 +53,10 @@ function semverCompare(version1: string, version2: string) {
 	return false;
 }
 
+function loggingResult(result: { stdout: string; stderr: string }) {
+	return result.stderr.length > 0 ? error(result.stderr) : info(result.stdout);
```

Comment:
I think we want to log both things, not just one or the other. My comment about making this conditional meant I don't want extra blank lines to be logged due to calling `error("")` when `result.stderr` is empty. We could do something like:
```ts
stdout && info(stdout)
stderr && error(stderr)
```

IMO, I wouldn't abstract just this logging logic into a separate function. It's not clear that this function performs a side-effect (which could be addressed with a different name), but more importantly the abstraction doesn't serve to simplify sufficiently complex logic to make the indirection worthwhile. it's roughly the same level of complexity, but with more ambiguity. it might make sense to wrap `execAsync`, however, to handle both executing a command and logging its result (including handling any resulting errors).